### ISSUE TITLE
Crash when mouse is moved over some affix items

### ISSUE THREAD

**Issue by bcodegard** [Shadows-of-Fire/Apotheosis#618]
When I moved my mouse over a particular affix item, my game consistently crashes. Just having the item in the inventory does not cause the crash, and neither does having it selected in the hotbar. I haven't been able to find any other items that cause this crash. For instance, other items with the omnetic affix have not causes crashes for me.

I managed to get the item data for the item that is causing the crash:
minecraft:diamond_pickaxe{Damage:0,affix_data:{affixes:{"apotheosis:destructive":0.8423403f,"apotheosis:lengthy":0.98316634f,"apotheosis:omnetic":0.26506817f},name:\'{"color":"#33FF33","translate":"%s %s %s","with":[{"translate":"affix.apotheosis:omnetic"},"",{"translate":"affix.apotheosis:lengthy.suffix"}]}\',rarity:"uncommon"},apoth_rchest:1b}

Attached is the error message produced.

[affix_item_crash_error.txt](https://github.com/Shadows-of-Fire/Apotheosis/files/9386487/affix_item_crash_error.txt)

**Comment by Shadowex3** [comment]
I think I hit the exact same error with a different pickaxe:

```
java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 3
	at shadows.apotheosis.adventure.affix.effect.OmneticAffix.addInformation(OmneticAffix.java:49) ~[Apotheosis-1.18.2-5.4.1.jar%2366!/:5.4.1] {re:classloading}
	at shadows.apotheosis.adventure.affix.AffixInstance.addInformation(AffixInstance.java:50) ~[Apotheosis-1.18.2-5.4.1.jar%2366!/:5.4.1] {re:mixin,re:classloading}
	at shadows.apotheosis.adventure.client.AdventureModuleClient.lambda$affixTooltips$3(AdventureModuleClient.java:170) ~[Apotheosis-1.18.2-5.4.1.jar%2366!/:5.4.1] {re:classloading}
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source) ~[?:?] {}
	at java.util.stream.SortedOps$SizedRefSortingSink.end(Unknown Source) ~[?:?] {}
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[?:?] {}
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[?:?] {}
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source) ~[?:?] {}
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source) ~[?:?] {}
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:?] {}
	at java.util.stream.ReferencePipeline.forEach(Unknown Source) ~[?:?] {}
	at shadows.apotheosis.adventure.client.AdventureModuleClient.affixTooltips(AdventureModuleClient.java:170) ~[Apotheosis-1.18.2-5.4.1.jar%2366!/:5.4.1] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_2251_AdventureModuleClient_affixTooltips_ItemTooltipEvent.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.event.ForgeEventFactory.onItemTooltip(ForgeEventFactory.java:273) ~[forge-1.18.2-40.1.73-universal.jar%23345!/:?] {re:mixin,re:classloading}
	at net.minecraft.world.item.ItemStack.m_41651_(ItemStack.java:768) ~[client-1.18.2-20220404.173914-srg.jar%23340!/:?]
```

**Comment by Sakreton** [comment]
I had the same issue, i could pinpoint it to an affixed iron shovel, sadly deleted the items before saving their names or tags.
For us the issue seamed to only happen when hovering over the item in an ME terminal, since we did not crash when we put the item into it.

[2022-08-21-1.log](https://github.com/Shadows-of-Fire/Apotheosis/files/9389354/2022-08-21-1.log)

**Comment by Shadows-of-Fire** [comment]
Fixed for 5.5.0
